.gitlab-ci-default-pipeline.yml

```diff
@@ -62,9 +62,10 @@ test:docs:
   before_script:
     - apt-get update && apt-get install -y
       docker.io bash git openssl pwgen python3 jq docker-compose wget sudo
+
     - git config --global user.name "user"
     - git config --global user.email "user@example.com"
-    - git clone https://github.com/mendersoftware/mender-docs.git mender-docs
+    - git clone https://gitlab.com/Northern.tech/Mender/mender-docs.git mender-docs
```

Comment:
This change is not necessary and adds a new entry point for error - if for any reason the GitLab fork is not up to date with GitHub, we might end up testing the wrong thing.

Reply:
Ok, now, after those discussions, I finally understood your comment in the related PR about "finding a way of testing it in CI"... It makes sense now. Thanks.